.gitignore

```diff
@@ -1,2 +1,5 @@
 _output
 release.yaml
+kubeconfig
```

Comment:
Is this something that our scripts/tooling generate? I would leave this out of a project .gitignore otherwise.

Reply:
To run the smoke-test we need to export the KUBECONFIG. its needed. Its mentioned in README